test/config/integration_tests.yml

```diff
@@ -171,6 +171,20 @@ testCases:
   launchCommand: nnictl view $resumeExpId
   experimentStatusCheck: False
 
+# Experiment import test
+- name: nnictl-experiment-import
+  configFile: test/config/nnictl_experiment/sklearn-classification.yml
+  setExperimentIdtoVar: $imoprtExpId
+  stopCommand: nnictl experiment import $imoprtExpId -f config/nnictl_experiment/test_import.json
+  onExitCommand: nnictl stop
+
+# Experiment export test
+- name: nnictl-experiment-export
+  configFile: test/config/nnictl_experiment/sklearn-classification.yml
+  launchCommand: nnictl resume $imoprtExpId
+  stopCommand: nnictl experiment export --filename config/nnictl_experiment/test_export.json --type json --intermediate
+  onExitCommand: nnictl stop
```

Comment:
This is not the proper way to use `stopCommand` and `onExitCommand`.
Here the result of `nnictl experiment export`  should be checked,  you can use a validator to run `export` command and validate the results.

Reply:
I will try to modify them and add a validator.